blockchain/system/constant.go

```diff
@@ -30,13 +30,18 @@ var (
 
 	// Canonical system contract names registered in Registry.
 	AddressBookName = "AddressBook"
+	GovParamName    = "GovParam"
+	Kip103Name      = "TreasuryRebalance"
 
 	AllContractNames = []string{
 		AddressBookName,
+		GovParamName,
+		Kip103Name,
 	}
 
 	// Some system contracts are allocated at special addresses.
```

Comment:
How about adding comments(maybe location?) about addresses for other system contracts? (GovParam, KIP103)

Reply:
Could you write a sample comment?

Reply:
I suggested adding a comment because someone might need clarification as to why there are no addresses except for reserved addresses. But I can't come up with the right sentence, let's skip it for now.